docs/Architecture.md

```diff
@@ -67,24 +67,31 @@ soon as it can, and all of the real work will be done as that user.
 
 ### Message
 
-### Policy
-
 ### Profile
 
->
-> TODO: Come up with a better name to distinguish it from *config profiles*.
->
+A profile is a configuration for Zonemaster Engine; see the [profiles
+overview] for context.
+Zonemaster Backend allows administrators to [configure the set of
+available profiles].
 
-### Config profile
+Each available profile has a [profile name].
+A profile named `default` is always available.
+Each available profile is based on the [Zonemaster Engine default profile].
+Each one (with the possible exception of `default`) has a [profile file]
```

Comment:
"[profile file]" --> "[profile files]" here or the reverse below.

Reply:
Updated the link reference definition instead.